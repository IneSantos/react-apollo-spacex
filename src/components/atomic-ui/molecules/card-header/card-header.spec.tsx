import React from "react";
import { render } from "@testing-library/react";
import CardHeader from "./card-header";

describe("CardHeader molecule tests", () => {
  test(`CardHeader component render`, () => {
    const { container } = render(<CardHeader name="This is a mission name"/>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
