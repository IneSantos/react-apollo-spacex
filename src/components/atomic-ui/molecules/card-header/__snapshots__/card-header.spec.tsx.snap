// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader molecule tests CardHeader component render 1`] = `
<div
  class="card-header"
>
  <h2
    class="title"
  >
    This is a mission name
  </h2>
</div>
`;
